config/samples/misf_v1alpha1_managedfusionoffering.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: misf.ibm.com/v1alpha1
+kind: ManagedFusionOffering
+metadata:
+  name: managedfusionoffering-sample
+  namespace: sample
+spec:
+  kind: DF
+  release: "4.10"
+  config: 
+    sample-key: sample-value
```

Comment:
An example should be representable. This one is meaningless as `sample-key` is a contrived key-value pair.
If you want to use `DF` as an example, use 4.12 and and a proper (syntactically) example

Reply:
Updated